src/components/ButtonComponents/NumberButtons/Numbers.js

```diff
@@ -1,16 +1,44 @@
-import React from "react";
+import React, { useState } from "react";
 
 //import any components needed
+import NumberButton from './NumberButton'
 
 //Import your array data to from the provided data file
+import { numbers } from '../../../data'
 
-const Numbers = () => {
+const Numbers = (props) => {
+  console.log(numbers)
   // STEP 2 - add the imported data to state
+  const [numberState, setNumberState] = useState(numbers)
+  console.log('Numbers', props)
+ 
   return (
-    <div>
-      {/* STEP 3 - Use .map() to iterate over your array data and return a button
-       component matching the name on the provided file. Pass
-       it any props needed by the child component*/}
-    </div>
-  );
+    // <div>  { numbers.map(number => <NumberButton key={number} text={number} addNumber={props.addNumber} />
+    // )}
+    // </div>
+    <>
+			<div>
+				{numbers.slice(0, 3).map(number => (
```

Comment:
I'm seeing that you're manually writing in the 4 rows of keys.  Can it be done easier using flexbox?